### DISCUSSION TITLE
Lesson7: Doubt in the syntax

### DISCUSSION THREAD

**Discussion by shivankar-p** [smartcontractkit/full-blockchain-solidity-course-py#890] [Q&A] [answered]
![Screenshot from 2022-01-29 15-04-14](https://user-images.githubusercontent.com/60961689/151655927-07de6848-2873-4479-b0a8-a0283fc90db7.png)
In line 60 of the code when I tried `assert lottery.lottery_state == 2` I was getting an error. Why do we need to add brackets to lottery_state to access it. i.e `assert lottery.lottery_state() == 2`. Isn't enum same as any other datatype?

**Comment by cromewar** [comment, marked as answer]
@shivankar-p Nice annotation , and yes the data type is the same, but this is way for brownie to understand you want to access to and object within the contract, so that's why we use ().
